Packs/WootCloud/Integrations/WootCloud/README.md

```diff
@@ -0,0 +1,465 @@
+Append HyperContext™ insights to your SIEM data and feed them into your orchestration workflows.
+This integration was integrated and tested with version xx of WootCloud
```

Comment:
add `command_examples` files and generate the documentation with the provided examples
https://xsoar.pan.dev/docs/integrations/integration-docs#run-demisto-sdk-generate-docs

https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/command_examples